guides/index.md

```diff
@@ -5,7 +5,33 @@ sidebar:
   nav: guides
 ---
 
-Want a guide instead of a map? Here's a collection of how-tos, tutorials and videos narrowly focused on a specific task you might want to accomplish. Entries are community-contributed, so please feel free to suggest a guide you'd like to add, or one you'd like to see others in the community put together.
+The number of Spinnaker guides is increasing incrementally, with contributions
```

Comment:
Reads a bit funny to have 'increasing' followed by 'incrementally'. Maybe something like "is continuing to increase", or just "is increasing"?